test/gtest/common/mem_buffer.h

```diff
@@ -7,10 +7,7 @@
 #ifndef GTEST_MEM_BUFFER_H_
 #define GTEST_MEM_BUFFER_H_
 
-#include <ucs/memory/memory_type.h>
-#include <stdint.h>
-#include <string>
-#include <vector>
```

Comment:
why remove? these are actually used by mem_buffer.h

Reply:
All of them present in `test.h`

Reply:
but mem_buffer.h does not include test.h
it's better each header be "self contained" and include all it needs by itself and not rely on including other headers before it

Reply:
It includes test.h on the next line https://github.com/openucx/ucx/pull/6859/files/3c024afd5de92074c1e23aa88541bbf80f3a484e#diff-d8746d5e7408ec94d471b40e0fc40e96260d03d0ec3c192302228290d90cae4cR10

you've told to include it, to have a macro for stdint.h

Reply:
right, sorry for the confusion